.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Proposed Changes
+
+
+### Checklist
+
+- [ ] I have bumped the necessary versions
+- [ ] I have (re-)packaged the updated charts `helm package $CHART_NAME -u`
+- [ ] I have updated the repo index `helm repo index .`
```

Comment:
Similarly, is there a .helmrc or something which indicates a minimum/max version of helm we need to use?